### DISCUSSION TITLE
Winpilot 3.3.40 Stargate ⊙🛸🚀

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#169] [Announcements]
### Important note: Bloatynosy/AI has now evolved into Winpilot. Check out [3.0.0](https://github.com/builtbybel/Winpilot/releases/tag/3.0.0) for more details 

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">My Winpilot is supported by Clippy 🛩️😂📎<a href="https://t.co/PkRbJxXfVg">https://t.co/PkRbJxXfVg</a> <a href="https://t.co/whQL0LHulG">pic.twitter.com/whQL0LHulG</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1772353175028187507?ref_src=twsrc%5Etfw">March 25, 2024</a></blockquote>

# Changelog
The **Assist me** feature down in the bottom right corner just got a major upgrade, now rolling with the code name **Stargate**. This handy helper walks you through customizing and debloating your Windows 10/11 system in just 4 simple steps. Additionally, the Stargate Assist has been optimized for debloating Legion GO and RogAlly devices.

![explorer_K8L0F4QNnm](https://github.com/builtbybel/Winpilot/assets/57478606/2e8bed5b-637f-4fcb-acf1-7160dd80043b)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/3.3.40'>Winpilot 3.3.40 Stargate ⊙🛸🚀</a>.</em>

**Comment by namhnz** [comment]
I can't type the Vietnamese language. Does it support other languages?
